woodwork/series_accessor.py

```diff
@@ -0,0 +1,120 @@
+            logical_type (LogicalType or str, optional): The logical type that should be assigned
+                to the series. If no value is provided, the LogicalType for the series will
+                be inferred. If the LogicalType provided or inferred does not have a dtype that
+                is compatible with the series dtype, an error will be raised.
+            semantic_tags (str or list or set, optional): Semantic tags to assign to the series.
+                Defaults to an empty set if not specified. There are two options for
+                specifying the semantic tags:
+                (str) If only one semantic tag is being set, a single string can be passed.
+                (list or set) If multiple tags are being set, a list or set of strings can be passed.
+            use_standard_tags (bool, optional): If True, will add standard semantic tags to the series
+                based on the inferred or specified logical type of the series. Defaults to True.
+            description (str, optional): Optional text describing the contents of the series.
+            metadata (dict[str -> json serializable], optional): Metadata associated with the series.
+        """
+        self._set_name(name)
+
+        logical_type = _get_column_logical_type(self._series, logical_type, self.name)
+
+        self._validate_logical_type(logical_type)
+
+        self._schema = _get_column_dict(name=self.name,
+                                        logical_type=logical_type,
+                                        semantic_tags=semantic_tags,
+                                        use_standard_tags=use_standard_tags,
+                                        description=description,
+                                        metadata=metadata)
+
+    @property
+    def description(self):
+        """The description of the series"""
+        return self._schema['description']
+
+    @description.setter
+    def description(self, description):
+        _validate_description(description)
+        self._schema['description'] = description
+
+    @property
+    def logical_type(self):
+        """The logical type of the series"""
+        return self._schema['logical_type']
+
+    @property
+    def metadata(self):
+        """The metadata of the series"""
+        return self._schema['metadata']
+
+    @metadata.setter
+    def metadata(self, metadata):
+        _validate_metadata(metadata)
+        self._schema['metadata'] = metadata
+
+    @property
+    def name(self):
+        """The name of the series"""
+        return self._series.name
+
+    @property
+    def schema(self):
+        return self._schema
+
+    @property
+    def semantic_tags(self):
+        """The semantic tags assigned to the series"""
+        return self._schema['semantic_tags']
+
+    def __eq__(self, other):
+        if self.schema != other.schema:
+            return False
+        return self._series.equals(other._series)
```

Comment:
Yeah, wasn't real sure how to handle this, but this seemed to make the most sense. I'm not sure how much this will actually get used though, but it didn't seem right to me to call two initialized accessors the same if the underlying data was different...